pyvista/utilities/reader.py

```diff
@@ -1115,7 +1143,9 @@ class POpenFOAMReader(OpenFOAMReader):
     parallel reconstructed data, and decomposed data.
     """
 
-    _class_reader = staticmethod(_vtk.lazy_vtkPOpenFOAMReader)
+    _class_reader = staticmethod(lazy_vtkPOpenFOAMReader)
+    _vtk_module_name = ''
+    _vtk_class_name = ''
```

Comment:
Do we need these lines?  They seem to reiterate the default setting.

```suggestion
```

Reply:
We do because we're overriding from the class we're inheriting from `OpenFOAMReader`.